### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by xathon**
Updated README.md to include holding down buttons and pressing keys in the web interface. I'll add a visual hint on the web interface.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,15 @@ This is also displayed via `--help`
 ```
 ### Web Interface
 Visit http://localhost:80 to use the WebInterface. Here you can control all supported Functions and read the Text from the LCD. 
+The following buttons can be held down to repeat the API calls: arrow up/down/left/right, lensshift up/down/left/right, focus near/far, zoom in/out.
+The following keys can be pressed:
```
Comment (localleon) [inline README.md]: Maybe the wording here is a little unclear. We could write something like `The following keys on your keyboard are shortcuts to these API/Button Functions `

**Comment by localleon** [comment]
Thanks for your contribution @xathon . I'm looking forward to the finished PR. Here are some suggestions

File: README.md (hunk 1 above)
Comment (xathon) [reply README.md]: Including this in the next commit.

Comment (xathon) [reply README.md]: Changed.

**Pull request merged**